src/queries.ts

```diff
@@ -0,0 +1,128 @@
+import { GQLEdgeInterface, GQLTransactionsResultInterface } from './gql_types';
+import fetch from 'node-fetch';
+
+const GQL_URL = 'https://arweave.net/graphql';
+const ITEMS_PER_REQUEST = 100;
+const VALID_APP_NAMES = ['ArDrive-Web', 'ArDrive-CLI', 'ArDrive-Sync'] as const;
+
+interface Query {
+	query: string;
+}
+
+export interface StakedPSTHolders {
+	[address: string]: number;
+}
+
+export async function getWaleltsEligibleForStreak(): Promise<StakedPSTHolders> {
```

Comment:
why `async` if you're not using `await`?

Reply:
510187db567602a4522f180595524524c23c4f7c